Jamfile

```diff
@@ -509,7 +509,7 @@ variant test_release : release
 	  <export-extra>on <debug-iterators>on <threading>multi
 	;
 variant test_debug : debug
-	: <crypto>openssl <logging>on <disk-stats>on
```

Comment:
leaving this feature on in this configuration is a good way to make sure it doesn't get broken in the future